util/site-dashboard/dashboard.js

```diff
@@ -113,6 +113,14 @@ class DashboardTable {
   render(node) {
     let table = document.createElement("table");
 
+    function make_header(level, text) {
+        let el_header = document.createElement(level);
+        let el_text = document.createTextNode(text);
```

Comment:
Couldn't this just be `el_header.textContent = text;`?

Reply:
Also, couldn't this just be a static part of HTML instead? It doesn't have to be inside the div.

Reply:
Good point! May as well not over-complicate it for now.

Made the headers static html.